docs: DOC-419 architecture section (#1026)
spectrocloud/librarium: content/docs/14-troubleshooting/06-nodes.md

---
title: "Nodes & Clusters"
metaTitle: "Troubleshooting steps for Kubernetes nodes and clusters"
metaDescription: "Troubleshooting steps for Kubernetes nodes and clusters when managed by Palette."
icon: ""
hideToC: false
fullWidth: false
---

# Nodes & Clusters

This page is covers common debugging scenarios for nodes and clusters post deployment.

# Nodes

## Scenario - Repaved Nodes

> Palette performs a rolling upgrade on the nodes when a chang is detected in the kubeadm config. Below are some of the actions including but not limited to that will cause the kubeadm configuration to change and result in the nodes getting an upgrade:

Reviewer: ```suggestion
Palette performs a rolling upgrade on the nodes when a change is detected in the kubeadm config. Below are some actions that can cause the kubeadm configuration to change and result in nodes getting upgraded:
```
